source/features/small-user-avatars.css

```diff
@@ -3,3 +3,7 @@
 	margin-left: -3px;
 	margin-right: 2px !important;
 }
+
+.user-mention .rgh-small-user-avatars {
+	margin-left: 1px;
+}
```

Comment:
Actually let's move this to the JSX instead. Then we can move the CSS rule above to `highlight-collaborators-and-own-conversations` instead and drop this CSS file altogether. You can:

- https://github.com/refined-github/refined-github/issues/6665

Reply:
766dba633eeeb7d9269ec0077e2f5ad9a07642b0